project-base/storefront/components/Blocks/SortingBar/SortingBarItem.tsx

```diff
@@ -1,16 +1,19 @@
-import { LabelLink } from 'components/Basic/LabelLink/LabelLink';
-import { twJoin } from 'tailwind-merge';
+import { twMergeCustom } from 'utils/twMerge';
 
 type SortingBarItemProps = { isActive: boolean; onClick?: () => void };
 
 export const SortingBarItem: FC<SortingBarItemProps> = ({ children, isActive, onClick }) => {
     return (
-        <LabelLink
-            className={twJoin('max-vl:bg-backgroundAccentLess max-vl:text-text', isActive && 'max-vl:text-textAccent')}
-            isActive={isActive}
+        <button
+            type="button"
+            className={twMergeCustom(
+                'text-right vl:text-center vl:relative vl:top-[0.5px] py-4 vl:py-2.5 vl:px-5 uppercase text-xs font-bold underline text-link hover:text-linkHovered vl:border vl:border-backgroundMore vl:bg-backgroundMore vl:rounded-t-xl vl:border-b-borderAccentLess',
+                isActive &&
+                    'font-semibold text-text no-underline vl:bg-background vl:border-borderAccentLess vl:border-b-background',
+            )}
             onClick={onClick}
         >
             {children}
-        </LabelLink>
+        </button>
     );
```

Comment:
I think it will better to get back to link. Some users maybe want to open new window with new ordering... when it is changed to button this availability disappears.

We spoke about it in chat.

Reply:
fixed :)